python/tank/api.py

```diff
@@ -221,6 +222,19 @@ def configuration_name(self):
     ##########################################################################################
     # public methods
 
+    def log_core_metric(self, action):
```

Comment:
I know this was a public method in the API spike, but now i am wondering if we should perhaps make it more internal? We are only intending this for internal use right? (like tracking when create_folders or path cache sync is run and that sort of thing). Maybe it could be moved into pipeline configuration which is not part of the official API. Or move it to the 'internal API' section above.